src/panels/alarm-table/AlarmTableControl.tsx

```diff
@@ -26,15 +26,37 @@ export const AlarmTableControl: React.FC<PanelProps<AlarmTableControlProps>> = (
     const { client } = useOpenNMSClient(props.data?.request?.targets?.[0]?.datasource)
     const { filteredProps, page, setPage, totalPages } = useAlarmProperties(props?.data?.series[0], props?.options?.alarmTable)
     const { table, menu, menuOpen, setMenuOpen } = useAlarmTableMenu(alarmIndexes, rowClicked, filteredProps, setState)
-    const { actions, detailsModal, setDetailsModal } = useAlarmTableMenuActions(state.indexes,
-      props?.data?.series?.[0]?.fields || [], () => setMenuOpen(false),
-      props?.options?.alarmTable?.alarmTableAdditional?.useGrafanaUser || false, client)
+
+    const { actions, detailsModal, setDetailsModal } = useAlarmTableMenuActions(
+      state.indexes,
+      props?.data?.series?.[0]?.fields || [],
+      () => setMenuOpen(false),
+      (actionName: string) => refreshPanel(actionName),
+      props?.options?.alarmTable?.alarmTableAdditional?.useGrafanaUser || false,
```

Comment:
Sonar would suggest using the `??` operator here, considering useGrafanaUser is a boolean. But the difference is almost negligible so just fyi.